src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -44,7 +50,25 @@ limitations under the License.
       <arg name="data" type="CHAR_STRING"/>
     </command>
 
+    <command source="server" code="0x01" name="StopAppResponse" optional="false">
+      <description>This command SHALL be generated in response to StopApp commands.</description>
+      <arg name="status" type="ApplicationLauncherStatus"/>
+      <arg name="data" type="CHAR_STRING"/>
+    </command>
+
+    <command source="server" code="0x02" name="HideAppResponse" optional="false">
+      <description>This command SHALL be generated in response to HideApp commands.</description>
+      <arg name="status" type="ApplicationLauncherStatus"/>
+      <arg name="data" type="CHAR_STRING"/>
+    </command>
+
   </cluster>
+
+  <struct name="ApplicationLauncherEndpoint">
```

Comment:
Where is this defined in the spec?

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/ApplicationLauncher.adoc#153-applicationep

Should be ApplicationEP, ApplicationEndpoint?